.eslintrc

```diff
@@ -4,20 +4,30 @@
     "strict": 0,
     "no-underscore-dangle": 0,
     "no-unused-vars": 0,
-    "curly": 0,
     "no-multi-spaces": 0,
     "key-spacing": 0,
```

Comment:
I haven't been consistent myself about that, but I do like the `"key-spacing": [2, { "align": "colon" }]` convention. Makes it imo easier to see what the values of an object are.